alacritty.yml

```diff
@@ -114,6 +110,10 @@ colors:
 
   # Colors the cursor will use if `custom_cursor_colors` is true
   cursor:
+    # Use custom cursor colors. If true, always display the cursor in the
+    # `colors.cursor.text` and `colors.cursor.cursor` colors,
+    # otherwise invert the colors of the background and foreground.
```

Comment:
:+1: Much better thank you. I guess we're sticking with `cursor.cursor`?

Reply:
I'm open for changing that. I don't really think it's that great, but I didn't have any idea that's clearly better.